### PR TITLE
add new version for rgi

### PR DESCRIPTION
**Pull request by raphenya**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by daler** [comment]
@raphenya what's the difference between this git repo and https://card.mcmaster.ca/download? Is it possible to use the latter? We archive tarballs in Galaxy's Cargo Port for long-term reproducibility, and using git urls prevents this.

File: recipes/rgi/3.2.1/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
```
Comment (daler) [inline recipes/rgi/3.2.1/build.sh]: These echos shouldn't be needed for conda recipes. You should be fine with just the `$PYTHON setup.py install`.

File: recipes/rgi/3.2.1/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+source:
+  git_url: {{ source_url }}
+  git_rev: {{ commit }}
+
+build:
+  skip: True # [py3k]
+  number: 1
+
+# We pin on gcc version to avoid gcc-5 dual ABI issues depending on boost.
```
Comment (daler) [inline recipes/rgi/3.2.1/meta.yaml]: this comment describes pinning gcc, but I don't see any pinning below, in which case you can remove this comment (if it doesn't really need to be pinned)

File: recipes/rgi/3.2.1/build.sh (hunk 1 above)
Comment (raphenya) [inline recipes/rgi/3.2.1/build.sh]: @daler thanks! would remove the echos.

File: recipes/rgi/3.2.1/meta.yaml (hunk 2 above)
Comment (raphenya) [inline recipes/rgi/3.2.1/meta.yaml]: @daler Thanks. Would remove comment.

**Comment by raphenya** [comment]
@daler there a way to use git urls,  since it's easier when importing other module to your project?

**Comment by daler** [comment]
@raphenya can you help explain the git repo you're using in this recipe? To me, it looks like the repo is a conda-specific wrapper (at least just based on the name) that wraps rgi proper. It does this by creating a git submodule that pulls in the rgi repo, and essentially installs the rgi scripts as part of a new rgi_conda_dev Python package.

Is that correct? If so, wouldn't it be easier to use the rgi repo directly, copying Python scripts to `$PREFIX/bin`?

Looks like the Python scripts don't have shebang lines, so you would have to add them in the conda recipe. Using the rgi release tarball from https://github.com/arpcard/rgi/archive/3.2.1.tar.gz, in build.sh you could do something like this:

```bash
#!/bin/bash
for fn in *py; do
  sed -i "1s|^|#!/usr/bin/env python\n|" $fn
  chmod +x $fn
  cp $fn $PREFIX/bin
done
```

Would this work for this recipe? It avoids coordinating a wrapper git repo plus submodule and uses a release tarball directly, resulting in easier maintenance in the long term.

**Comment by raphenya** [comment]
@daler yes the repo is a wrapper for rgi.

**Pull request merged**